cmd/ndm_daemonset/controller/probe.go

```diff
@@ -101,7 +101,6 @@ func (c *Controller) ListProbe() []*Probe {
 // FillBlockDeviceDetails lists registered probes and fills details from each probe
 func (c *Controller) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevice) {
 	blockDevice.NodeAttributes = c.NodeAttributes
-	blockDevice.DeviceAttributes.DeviceType = NDMDefaultDiskType
```

Comment:
why do need to remove DeviceType? Pleae check if there is any component dependant on it.

Reply:
nvm, I read it wrong.